tests/functional/test_deploy.py

```diff
@@ -54,19 +55,21 @@ def source(request):
     await model.block_until(lambda: app.status == "waiting")
 
 
+@pytest.mark.timeout(30)
 async def test_redis_deploy(model, series, app, request):
     """Create a Redis deployment for testing relation to GitLab."""
     if app.name.endswith("jujucharms"):
         pytest.skip("No need to test the charm deploy")
 
     application_name = "gitlab-redis-{}".format(series)
-    cmd = "juju deploy cs:~omnivector/redis -m {} --series xenial {}".format(
+    cmd = "juju deploy cs:~redis-charmers/redis -m {} --series xenial {}".format(
```

Comment:
This seems like it should be in #6 - or does the redis change need to be merged before this one?

Reply:
The redis change has to land first, w/o that branch functional tests fail and there is a functional test for the backup action that I'd like to see passing.